geex-core/src/main/java/nl/tudelft/context/drawable/graph/AbstractLabel.java

```diff
@@ -9,16 +9,4 @@
  */
 public abstract class AbstractLabel extends VBox {
 
-    /**
-     * Draw sub elements when needed.
-     */
-    public abstract void init();
-
-    /**
-     * Get the current column the label is displayed.
-     *
-     * @return Column index
-     */
-    public abstract int currentColumn();
-
```

Comment:
Is `AbstractLabel` still relevant when it is empty? Now you can just say `VBox label = new DrawableGraphNodeLabel(...);`